synfig-studio/ui/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ set(UI_FILES
     canvas_options.glade
     preview_options.glade
     dialog_workspaces.glade
+	shortcuts_window.glade
```

Comment:
indentation?

Reply:
Kept for last so code review would be easier :) Fixing indentation early makes the actual changes hard to investigate don't you think?